test/models/page.test.js

```diff
@@ -279,4 +283,40 @@ describe('Page', function () {
     });
   });
 
+  describe('.findPage', function () {
+    context('findPageById', () => {
+      it('should find page', function(done) {
+        const pageToFind = createdPages[0];
+        Page.findPageById(pageToFind._id)
+        .then(pageData => {
+          expect(pageData.path).to.equal(pageToFind.path);
+          done();
+        });
+      });
+    });
+
+    context('findPageByIdAndGrantedUser', function() {
+      it('should find page', function(done) {
+        const pageToFind = createdPages[0];
+        const grantedUser = createdUsers[0];
+        Page.findPageByIdAndGrantedUser(pageToFind._id, grantedUser)
+        .then(pageData => {
+          expect(pageData.path).to.equal(pageToFind.path);
+          done();
+        });
+      });
+
+      it('should error by grant', done => {
+        const pageToFind = createdPages[0];
+        const grantedUser = createdUsers[1];
+        Page.findPageByIdAndGrantedUser(pageToFind._id, grantedUser)
+        .then(pageData => {
```

Comment:
This line is unnecessary.

Reply:
Added done with error below. 
(Because the test should be failed if the user found)